Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    float speed = 10f;
+    float jumpForce = 10f;
```

Comment:
It's a good idea to serialize these fields since these are things you want to be able to adjust as a designer!